llvm/test/CodeGen/X86/bfloat-constrained.ll

```diff
@@ -0,0 +1,180 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
+; RUN: llc < %s -mtriple=i686-linux-gnu | FileCheck %s --check-prefix=X32
```

Comment:
Please X86 instead of X32. X32 is used for x32 mode.

Reply:
Borrowed from `llvm/test/CodeGen/X86/half-constrained.ll` as well. 😄

Reply:
Solved by https://github.com/llvm/llvm-project/commit/ccc48d45b832def14c8bc1849cf64c805892368d :)